packages/editor-ui/src/components/ExecutionsUsage.vue

```diff
@@ -93,43 +87,40 @@ export interface Trial {
 	length: number;
 	gracePeriod: number;
 }
-export interface Instance {
-	createdAt: string;
-}
+
 export interface Usage {
 	executions: number;
 	activeWorkflows: number;
 }
 
 const props = defineProps<{ cloudPlanData: CloudPlanData }>();
 
-const now = DateTime.utc();
+const emit = defineEmits(['onUpgradePlanClicked']);
 
-onMounted(async () => {
-	console.log('monte');
-});
+const now = DateTime.utc();
 
-const daysLeftOnTrial = computed(() => {
+const daysLeftOnTrial = () => {
 	const { days = 0 } = getPlanExpirationDate().diff(now, ['days']).toObject();
 	return Math.ceil(days);
-});
+};
 
-const isTrialExpired = computed(() => {
-	const trialEndsAt = DateTime.fromISO(props.cloudPlanData.planSpec.expirationDate);
+const isTrialExpired = () => {
+	const trialEndsAt = DateTime.fromISO(props.cloudPlanData.expirationDate);
 	return now.toMillis() > trialEndsAt.toMillis();
-});
+};
 
-const getPlanExpirationDate = () => DateTime.fromISO(props.cloudPlanData.planSpec.expirationDate);
+const getPlanExpirationDate = () => DateTime.fromISO(props.cloudPlanData.expirationDate);
```

Comment:
sorry I guess it was cleaner before with `computed`.. I thought maybe falsely that it should be reactive without that